plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -85,6 +87,13 @@ private boolean validateLocalFields() {
       return false;
     }
     
+    File parent = getLocationPath().toFile();
+    File projectDirectory = new File(parent, getProjectName());
```

Comment:
A project's name is taken from the .project file, and it can differ from the project directory.